lib/models/os.ts

```diff
@@ -21,15 +21,9 @@ import once = require('lodash/once');
 import * as memoizee from 'memoizee';
 import * as semver from 'semver';
 
-import {
-	deviceTypes as deviceTypesUtil,
-	isDevelopmentVersion,
-	isNotFoundResponse,
-	onlyIf,
-	treatAsMissingApplication,
-} from '../util';
-
-import { hupActionHelper } from '../util/device-actions/os-update/utils';
+import { isNotFoundResponse, onlyIf, treatAsMissingApplication } from '../util';
+import * as deviceTypesUtils from '../util/device-types';
```

Comment:
B/c `isOsArchitectureCompatibleWith` is re-exported by this module there is no point of lazy loading this.